### ISSUE TITLE
Charting technology?

### ISSUE THREAD

**Issue by derrickp** [derrickp/exercise-map#3]
What should we be doing for charting? Should we build our own charting SDK or use one that already exists in Android?

**Comment by sutherlanda** [comment]
It looks like there are already quite a few Charting SDKs available for Android (http://stackoverflow.com/questions/9741300/charts-for-android).  I think it would probably be worthwhile to do some investigation on which of these libraries would best suit our needs.  I'll start poking through some of the libraries from that list (starting with the free ones) in order to get a better idea.  I'm also open to the idea of developing our own SDK though, as that would be super fun.

**Comment by derrickp** [comment]
Yeah I think taking a look at what's already available would be cool. While
I think it'd be great to do our own charting SDK we already have a lot to
think about with the map control alone.

I got excited for Telerik until I saw it was $500. Which is a lot when
you're just starting out.
